tests/test_snappy_itransfer_step.py

```diff
@@ -0,0 +1,156 @@
+"""Tests for ``cubi_tk.snappy.itransfer_step``.
+
+We only run some smoke tests here.
+"""
+
+import os
+from unittest import mock
+from unittest.mock import ANY
+
+import pytest
+from pyfakefs import fake_filesystem
+
+from .conftest import my_exists, my_get_sodar_info
+from cubi_tk.__main__ import setup_argparse, main
+
+
+def test_run_snappy_itransfer_ngs_mapping_help(capsys):
+    parser, subparsers = setup_argparse()
+    with pytest.raises(SystemExit) as e:
+        parser.parse_args(["snappy", "itransfer-step", "--help"])
+
+    assert e.value.code == 0
+
+    res = capsys.readouterr()
+    assert res.out
+    assert not res.err
+
+
+def test_run_snappy_itransfer_ngs_mapping_nothing(capsys):
```

Comment:
If you agree to the `None` suggestion for the default, perhaps you can include a new test to assert that the exit code is as expected.